contracts/zero-ex/contracts/src/features/libs/LibSignature.sol

```diff
@@ -44,7 +44,8 @@ library LibSignature {
         ILLEGAL,
         INVALID,
         EIP712,
-        ETHSIGN
+        ETHSIGN,
+        PRESIGNED
```

Comment:
This has a different value than in v3, idk if that'll be a problem

Reply:
could pad this enum with `WALLET` and `VALIDATOR` so everything is aligned with v3